src/main/java/ArrayStack.java

```diff
@@ -19,15 +17,15 @@ public ArrayStack() {
     /**
      * Create an ArrayStack with the specified size.
      *
-     * @param size      Starting size of the fixed length array.
+     * @param initialCapacity Starting capacity of the fixed length array.
      */
     @SuppressWarnings("unchecked")
-    public ArrayStack(int size) {
+    public ArrayStack(int initialCapacity) {
```

Comment:
It's also the [ArrayList terminology](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/ArrayList.html#%3Cinit%3E(int)).

Reply:
Happy accident.